File: android/app/src/main/java/com/connect/club/react/bridge/iosLike/RemoteParticipant.kt
package com.connect.club.react.bridge.iosLike

import org.webrtc.RtpSender

class RemoteParticipant(val endpoint: String) {

    var videoTrack: VideoTrack? = null
    var rtpSender: RtpSender? = null
    var audioTrack: AudioTrack? = null
    var capturer: LocalVideoCapturer? = null

    fun destroy() {
        capturer?.destroy()
        capturer = null
    }

    override fun toString(): String {
        return "endpoint='$endpoint', videoTrack=${videoTrack?.track?.id()}"
    }
}

File: ios/JitsiWebRTC/jitsi-webrtc/Models/RemoteParticipant.swift
//
//  RemoteParticipant.swift
//  jitsi-webrtc
//
//  Created by Тарас Минин on 30/07/2020.
//  Copyright © 2020 CNNCT Limited. All rights reserved.
//

import Foundation
import WebRTC

public final class RemoteParticipant: NSObject {
    public let endpoint: String

    public var videoTrack: RTCVideoTrack?
    public var rtcRtpSender: RTCRtpSender?
    public var audioTrack: RTCAudioTrack?
    public var capturer: LocalVideoCapturer?

    public init(endpoint: String) {
        self.endpoint = endpoint
        super.init()
    }
}
